### ISSUE TITLE
[BUG] False positive in `no-contradicting-classname` rule for `break-words` + `break-all`

### ISSUE THREAD

**Issue by rider159159** [francoismassart/eslint-plugin-tailwindcss#440]
## Describe the bug

The `no-contradicting-classname` rule incorrectly flags `break-words` and `break-all` as contradicting classnames. However, these two utilities apply to **different CSS properties** and are not conflicting:

- `.break-words` → `overflow-wrap: break-word`
- `.break-all` → `word-break: break-all`

Using them together is intentional and valid CSS — they control separate aspects of text breaking behavior.

## To Reproduce

```html
<p class="break-words break-all">Some long text content</p>
```

**ESLint output:**
```
warning  Classnames break-words, break-all are conflicting!  tailwindcss/no-contradicting-classname
```

## Expected behavior

No warning should be raised. These two classes target different CSS properties (`overflow-wrap` vs `word-break`) and do not conflict.

## Actual Tailwind CSS output

```css
/* break-words */
.break-words {
  overflow-wrap: break-word;
}

/* break-all */
.break-all {
  word-break: break-all;
}
```

Different properties — no contradiction.

## Environment

- OS: Windows 11
- Node.js: v24.1.0
- eslint: v9.39.2
- eslint-plugin-tailwindcss: v3.18.2
- tailwindcss: v3.4.4

## ESLint config

```js
// eslint.config.mjs
{
  rules: {
    'tailwindcss/no-contradicting-classname': 'warn',
  }
}
```

## Additional context

Related to other known false positives in this rule:
- #271 (custom gradient values)
- #164 (arbitrary variants)
- #14 (`grid grid-*` classes)

The rule appears to group all `break-*` utilities into a single "contradicting" group without checking whether they actually share the same CSS property.

**Comment by francoismassart** [comment]
Hi @rider159159 
Thank you for reporting the issue.

Bad news, I'll not work anymore on v3

Good news, the bug is fixed on v4, I added your test case

<img width="777" height="347" alt="Image" src="https://github.com/user-attachments/assets/b1cd9ea2-b2c8-4441-bbca-dae5912ed65f" />

V4 is almost ready and you can already try it out via:

```
npm i eslint-plugin-tailwindcss@4.0.0-alpha.5
```
